ckanext/switzerland/templates/activity_streams/activity_stream_email_notifications.text

```diff
@@ -1,5 +1,5 @@
-{% set num = activities|length %}{{ ungettext("You have {num} new activity on your {site_title} dashboard", "You have {num} new activities on your {site_title} dashboard", num).format(site_title=g.site_title, num=num) }} {{ _('To view your dashboard, click on this link:') }}
+{% set num = activities|length %}{{ ngettext("You have {num} new activity on your {site_title} dashboard", "You have {num} new activities on your {site_title} dashboard", num).format(site_title=g.site_title if g else site_title, num=num) }} {{ _('To view your dashboard, click on this link:') }}
```

Comment:
I think `ngettext` is a typo?

Reply:
sorry, I meant to leave a comment about that and forgot. It really is `ngettext`, and there are both functions in Jinja2. Not sure what the actual difference is, though. :sweat_smile:

Reply:
ah, thanks for clarifying :D